docs/api/node.md

```diff
@@ -155,6 +165,38 @@ interface Firmware {
 
 If no firmware data can be extracted, the method will throw.
 
+Example usage:
+
+```ts
+// Extract the firmware from the uploaded file
+const rawData = Buffer.from(firmware);
```

Comment:
Hmm I think this is confusing if you read this the first time. Maybe just state that `rawData` is a Buffer with the file contents and `filename` the corresponding filename?

Reply:
Yep done